mambaSharedFramework/Pantos-Generic Playlist Parsing/Pantos-Generic Tag Writers/GenericSingleTagWriter.swift

```diff
@@ -39,7 +39,7 @@ public struct GenericSingleTagWriter: HLSTagWriter {
         }
         
         try stream.write(stringRef: tag.tagName!)
-        try stream.write(unicodeScalar: HLSTagWritingSeparators.colon)
+        try stream.write(unicodeScalar: TagWritingSeparators.colon)
```

Comment:
Observation, `TagWritingSeparators` is one of the few places where `HLS` isn't replaced by `Playlist` (e.g., `PlaylistTagWritingSeparators`)